src/schema/v2/partner/partner.ts

```diff
@@ -470,6 +470,10 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
 
               return {
                 totalCount,
+                pageCursors: createPageCursors(
+                  { ...args, page, size },
+                  totalCount
+                ),
```

Comment:
#non-blocking - if you feel like making this modern, this is how we do this nowadays: https://github.com/artsy/metaphysics/blob/main/docs/pagination.md

should be a simple change, but feel free to ignore

Reply:
Yea, just use that helper!

Reply:
Thanks for letting me know! I'll go ahead and update it as there is no time constraint for this work